diff --git a/containerd-template.toml b/containerd-template.toml
index a5a73e8..846816d 100644
--- a/containerd-template.toml
+++ b/containerd-template.toml
@@ -43,14 +43,18 @@ oom_score = 0
     no_pivot = false

     # default_runtime_name is the default runtime name to use.
-    default_runtime_name = "${RUNTIME}"
+    default_runtime_name = "crun"

     # 'plugins."io.containerd.grpc.v1.cri".containerd.runtimes' is a map from CRI RuntimeHandler strings, which specify types
     # of runtime configurations, to the matching configurations.
     # In this example, 'runc' is the RuntimeHandler string to match.
-    [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+    [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.crun]
+      pod_annotations = ["*.wasm.*", "wasm.*", "module.wasm.image/*", "*.module.wasm.image", "module.wasm.image/variant.*"]
       # runtime_type is the runtime type to use in containerd e.g. io.containerd.runtime.v1.linux
-      runtime_type = "${RUNTIME_TYPE}"
+      runtime_type = "io.containerd.runc.v2"
+
+      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.crun.options]
+        BinaryName = "/usr/local/bin/crun"

     [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.nvidia-container-runtime]
       # runtime_type is the runtime type to use in containerd e.g. io.containerd.runtime.v1.linux
